app/gateway/network-ports-firewall.md

```diff
@@ -1,25 +1,62 @@
 ---
-title: Network, ports, and firewall for {{site.base_gateway}}
+title: "{{site.base_gateway}} ports"
 content_type: reference
 layout: reference
 
 products:
     - gateway
 
+min_version:
+  gateway: '3.6'
+
 description: placeholder
 
 related_resources:
   - text: "Secure {{site.base_gateway}}"
     url: /gateway/security/
+  - text: Proxying with {{site.base_gateway}}
+    url: /gateway/traffic-control/proxying/
+  - text: "{{site.base_gateway}} networking"
+    url: /gateway/network/
 ---
 
-@todo
+{{site.base_gateway}} needs port access for two main types of connections: traffic passing through the proxy and managing the {{site.base_gateway}} via the Admin API.
+
+## Proxy ports
+
+In general, the proxy ports are the *only* ports that should be made available to your clients. Upstream services are accessible via the proxy interface and ports, so make sure that these values only grant the access level you require. 
+
+Your proxy will need have rules added for any HTTP/HTTPS and TCP/TLS stream listeners that you configure. For example, if you want {{site.base_gateway}} to manage traffic on port `4242`, your firewall must allow traffic on that port.
+
+The following are the default proxy ports:
+
+| Port | Protocol | `kong.conf` setting | Description | 
+|---------|---------|------------|------------|
+| `8000` | HTTP     | [`proxy_listen`](/gateway/configuration/) | Takes incoming HTTP traffic from [Consumers](/gateway/entities/consumer/), and forwards it to upstream services. | 
+| `8443` | HTTPS    | [`proxy_listen`](/gateway/configuration/) | Takes incoming HTTPS traffic from [Consumers](/gateway/entities/consumer/), and forwards it to upstream services. |
```

Comment:
```suggestion
| `8000` | HTTP     | [`proxy_listen`](/gateway/configuration/#proxy_listen) | Takes incoming HTTP traffic from [Consumers](/gateway/entities/consumer/), and forwards it to upstream services. | 
| `8443` | HTTPS    | [`proxy_listen`](/gateway/configuration/#proxy_listen) | Takes incoming HTTPS traffic from [Consumers](/gateway/entities/consumer/), and forwards it to upstream services. |
```
The params for these anchors don't exist yet, but they will, and they won't break the link checker.